src/main/java/org/jenkinsci/remoting/engine/JnlpAgentEndpointResolver.java

```diff
@@ -317,6 +320,9 @@ public void waitForReady() throws InterruptedException {
                     LOGGER.log(Level.INFO,
                             "Master isn't ready to talk to us on {0}. Will retry again: response code={1}",
```

Comment:
Not introduced here, but still: `retry again` is redundant. You either *try again*, or you *retry*.

Reply:
redundancy re-corrected